test/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests/OpenTelemetry.Instrumentation.EntityFrameworkCore.Tests.csproj

```diff
@@ -6,7 +6,7 @@
     <Description>Unit test project for OpenTelemetry Microsoft.EntityFrameworkCore instrumentation.</Description>
   </PropertyGroup>
 
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net8.0' ">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net8.0'">
```

Comment:
nit: Change this one to `'$(TargetFrameworkIdentifier)' == '.NETCoreApp'` so it doesn't have to be changed if something like `net9.0` is ever added.

Reply:
The package `Microsoft.EntityFrameworkCore.Sqlite` only targets `net8.0` 😔. When future FTMs will be added, the package version will need to be reviewed here I think.

Reply:
If you target `net9.0` and reference `Microsoft.EntityFrameworkCore.Sqlite` it will use the `net8.0` bits and that is totally fine 😄

Reply:
I'm probably tired, I saw it in the other way around. It's working in this direction indeed 😄 I'll address the feedback, thank you!